[error]Unhandled: Path must be a string. Received undefined

```
Starting: NuKeeper
==============================================================================
Task         : NuKeeper
Description  : Version 0.4.34
Version      : 0.4.34
Author       : NuKeeper
Help         : For help please visit https://github.com/NuKeeperDotNet/NuKeeper
==============================================================================
##[error]Unhandled: Path must be a string. Received undefined
Finishing: NuKeeper
```
Here are my settings, is any of this incorrect?:

![image](https://user-images.githubusercontent.com/3595932/59769879-48cb7d80-92a7-11e9-8d71-f31bea1692ab.png)

I am still getting the error, it seems unrelated to my config, it seems to be an error with the task metadata?

```
##[debug]Evaluating condition for step: 'NuKeeper' |  
-- | --
  | ##[debug]Evaluating: succeeded() |  
  | ##[debug]Evaluating succeeded: |  
  | ##[debug]=> True |  
  | ##[debug]Result: True |  
  | Starting: NuKeeper |  
  | ============================================================================== |  
  | Task         : NuKeeper |  
  | Description  : Version 0.4.34 |  
  | Version      : 0.4.34 |  
  | Author       : NuKeeper |  
  | Help         : For help please visit https://github.com/NuKeeperDotNet/NuKeeper |  
  | ============================================================================== |  
  | ##[debug]agent.TempDirectory=D:\9\_work\_temp |  
  | ##[debug]loading inputs and endpoints |  
  | ##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN |  
  | ##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION |  
  | ##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION |  
  | ##[debug]loading INPUT_ARGUMENTS |  
  | ##[debug]loading INPUT_CHECKLATEST |  
  | ##[debug]loading INPUT_TARGETFOLDER |  
  | ##[debug]loading INPUT_VERSION |  
  | ##[debug]loading SECRET_NUKEEPER_AZURE_DEVOPS_TOKEN |  
  | ##[debug]loaded 8 |  
  | ##[debug]Agent.ProxyUrl=undefined |  
  | ##[debug]Agent.CAInfo=undefined |  
  | ##[debug]Agent.ClientCert=undefined |  
  | ##[debug]Agent.SkipCertValidation=undefined |  
  | ##[debug]Agent.ProxyUrl=undefined |  
  | ##[debug]Agent.CAInfo=undefined |  
  | ##[debug]Agent.ClientCert=undefined |  
  | ##[debug]check path : D:\9\_work\_tasks\NuKeeper_e5a17a66-903f-46a9-a9b9-94241bcedc39\0.4.34\node_modules\azure-pipelines-tool-lib\lib.json |  
  | ##[debug]adding resource file: D:\9\_work\_tasks\NuKeeper_e5a17a66-903f-46a9-a9b9-94241bcedc39\0.4.34\node_modules\azure-pipelines-tool-lib\lib.json |  
  | ##[debug]system.culture=en-US |  
  | ##[debug]Pipeline.Workspace=undefined |  
  | ##[debug]task result: Failed |  
  | ##[error]Unhandled: Path must be a string. Received undefined |  
  | ##[debug]Processed: ##vso[task.issue type=error;]Unhandled: Path must be a string. Received undefined |  
  | ##[debug]Processed: ##vso[task.complete result=Failed;]Unhandled: Path must be a string. Received undefined |  
  | Finishing: NuKeeper
```

Settings:

![image](https://user-images.githubusercontent.com/3595932/60510445-b2984e00-9ccf-11e9-98d3-385262c61d18.png)

--- a/nukeeper/src/FeedInstaller.ts
+++ b/nukeeper/src/FeedInstaller.ts
@@ -13,7 +13,11 @@ export interface IPackageSource {
     feedUri: string;
 }
 
-const nugetConfigPath = path.join(tl.getVariable('Pipeline.Workspace'), "NuGet.config");
+const workspace = tl.getVariables().some(x => x.name === 'Pipeline.Workspace') ? 
+    tl.getVariable('Pipeline.Workspace') : 
+    tl.getVariable('Agent.BuildDirectory');
+    
+const nugetConfigPath = path.join(workspace, "NuGet.config");
 
 export default async function createFeed(inputParam: string, nukeeperPath: string): Promise<any> {
     try {

